s3watcher/lib/Upload.js

```diff
@@ -0,0 +1,38 @@
+var request = require('request');
+var Rx = require('rx');
+
+
+module.exports = {
+    buildUpload: function(config, s3Event) {
+         const headers = {
+            'Content-Length': s3Event.size,
+            'Content-Type': 'application/octet-stream',
+            'X-Gu-Media-Key': config.apiKey
+        };
+
+        return  {
+            key: config.apiKey,
+            url: config.baseUrl + "/images?filename=" + s3Event.filename,
```

Comment:
Could the filename be passed along as a `params` to the `request` call, to ensure proper encoding of it?

We will need to send the `uploadedBy` information too (folder?).